python_modules/libraries/dagster-embedded-elt/dagster_embedded_elt/sling/resources.py

```diff
@@ -95,8 +96,127 @@ class SlingTargetConnection(PermissiveConfig):
     )
 
 
+class SlingSyncBase:
```

Comment:
Would add a docstring clarifying this class as well. probably worth making it @experimental

Reply:
`SlingSyncBase` isn't publicly exposed, should we still mark it experimental and document it?

Reply:
If it's not a public class it should be marked `_SlingSyncBase`, but if you expect people might want to implement on top of this class then I would call add @experimental. 

In both cases, the docstring is helpful to explain the class's purpose to developers. I don't think you need the full API documentation with types/parms if its not public but 1-2 lines would be enough.